package.json

```diff
@@ -1,11 +1,15 @@
 {
   "name": "snyk-php-plugin",
   "description": "Snyk CLI PHP plugin",
-  "main": "lib/index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "lint": "eslint -c .eslintrc lib test",
-    "test": "tap test/*.test.js -R spec",
-    "semantic-release": "semantic-release"
+    "build": "tsc",
+    "build-watch": "tsc -w",
+    "lint": "tslint -p tsconfig.json",
```

Comment:
TSLint is being deprecated https://github.com/palantir/tslint/issues/4534 . ESlint will be the standard linter for javascript and typscript. Should we be using tslint for now or leave as ESlint?

Reply:
@lwywoo, I wouldn't use it for now. But we should keep that in mind and plan for next releases.

Reply:
@lwywoo, cool! Will apply this then. Thanks for spotting this out